addons/website_sale_slides/static/src/js/slides_course_join.js

```diff
@@ -36,6 +36,7 @@ CourseJoinWidget.include({
                     },
                     {
                         isBuyNow: true,
+                        showQuantity: Boolean(ev.currentTarget.dataset.showQuantity),
```

Comment:
Is it needed here? Is there a way to open a combo or a product configurator for an online course?